README.md

```diff
@@ -49,4 +37,10 @@ You can tweak the log output by setting the following environment variables:
 
 - `LOGFORMAT` a go template string for the generated logs
 - `FREQUENCY` the maximum delay between log messages
-- `CONSTANTRATE` by default the log generator will output messages ever 0 - `FREQUENCY` seconds.  If this variable is set to true messages will be logged ever `FREQUENCY` seconds.
+- `CONSTANTRATE` by default the log generator will output messages every - `FREQUENCY` seconds.  If this variable is set to true messages will be logged every `FREQUENCY` seconds.
+
+## Log Output
+
+`docker run -p 80:80 paynejacob/log-output:latest`
+
+The log output is a simple http service that can receive logs and be used to test a logging setup. A `GET /` will return a summary of all logs received by the service. Logs are organized into indices.  Think of an index as a folder for logs. The logs are written by preforming `POST /<index name>/` with one or more log messages.  A `GET /<index name>/` will return the last 100 logs received by the service for the given index. **Logs are stored in memory and all statistics and logs will be lost if the pod is restarted.**

```

Comment:
nit: "Logs are organized into indices, which can be thought of as folders/directories for logs."